roles/pulp-webserver/templates/pulp-vhost.conf.j2

```diff
@@ -37,4 +37,7 @@ Define pulp-api {{ pulp_api_host }}:{{ pulp_api_port }}
 
   IncludeOptional pulp/*.conf
```

Comment:
It only works if I remove the `.conf`
I believe it doesn't handle well 2 symlinks with same name: `apache.conf`:
```
[vagrant@pulp3-source-fedora30 ~]$ ls -al /etc/httpd/pulp/
total 16
drwxr-xr-x. 2 root root 4096 Apr 24 19:30 .
drwxr-xr-x. 6 root root 4096 Apr 24 19:30 ..
lrwxrwxrwx. 1 root root   74 Apr 24 19:30 galaxy_ng -> /home/vagrant/devel/galaxy_ng/galaxy_ng/app/webserver_snippets/apache.conf
lrwxrwxrwx. 1 root root   80 Apr 24 19:30 pulp_ansible -> /home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/apache.conf
```

Reply:
oh yeah that's a problem